src/realm/sync/protocol.hpp

```diff
@@ -258,22 +262,24 @@ struct ProtocolErrorInfo {
         ClientReset,
         ClientResetNoRecovery,
         MigrateToFLX,
-        RevertToPBS
+        RevertToPBS,
+        RefreshUser,
```

Comment:
Does the server send these two new actions? Or is it something we set internally? If it's the latter, maybe consider adding a comment.

Reply:
Right now it's something we set internally, but the server could send it eventually.